build/schema.md

```diff
@@ -0,0 +1,79 @@
+# Schema
+
+The value for `state` properties. Defines the default state property value, and its type.
+
+The schema definition follows a subset of [JSON Schema](https://json-schema.org/) but is more strict in terms of what properties are required.
```

Comment:
See me other comment about using a different "language".

Reply:
"schema" keywords were removed from the docs, https://github.com/maplibre/maplibre-style-spec/pull/1044#discussion_r2024346465